docs/configuration.rst

```diff
@@ -31,6 +31,8 @@ The following environment variables for the tracer are supported:
    DD_TAGS:
      description: |
          Set global tags to be attached to every span. Value must be either comma or space separated. e.g. ``key1:value1,key2:value2`` or ``key1:value key2:value2``.
+
+         If a tag value is not supplied the key will be set as a tag. e.g. ``key_val1,key_val2`` is equivalent to ``key_val1:key_val1,key_val2:key_val2``.
```

Comment:
Is that true?
It should be `key_val1:http.request.headers.key_val1`, see first couple examples under "Examples" - https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2302444638/DD+TRACE+HEADER+TAGS#Examples